File: app/src/test/java/com/fallllllll/lipperwithkotlin/MyRobolectricTestRunner.java
package com.fallllllll.lipperwithkotlin;

import org.junit.runners.model.InitializationError;
import org.robolectric.RoboSettings;
import org.robolectric.RobolectricTestRunner;

/**
 * Created by fallllllll on 2017/3/9.
 */

public class MyRobolectricTestRunner extends RobolectricTestRunner {
    /**
     * Creates a runner to run {@code testClass}. Looks in your working directory for your AndroidManifest.xml file
     * and res directory by default. Use the {@link Config} annotation to configure.
     *
     * @param testClass the test class to be run
     * @throws InitializationError if junit says so
     */
    public MyRobolectricTestRunner(Class<?> testClass) throws InitializationError {
        super(testClass);
        RoboSettings.setMavenRepositoryId("alimaven");
        RoboSettings.setMavenRepositoryUrl("http://maven.aliyun.com/nexus/content/groups/public/");
    }

}

File: app/src/test/java/com/fallllllll/lipperwithkotlin/core/MyRobolectricTestRunner.kt
package com.fallllllll.lipperwithkotlin.core

import org.robolectric.RoboSettings
import org.robolectric.RobolectricTestRunner

/**
 * Created by fallllllll on 2017/6/20/020.
 * GitHub :  https://github.com/348476129/Lipper
 */
class MyRobolectricTestRunner(testClass:Class<Any>) : RobolectricTestRunner(testClass) {
    init {
        RoboSettings.setMavenRepositoryId("alimaven")
        RoboSettings.setMavenRepositoryUrl("http://maven.aliyun.com/nexus/content/groups/public/")
    }
}